src/stories/containers/EndgamePhaseOneProgress/components/RoadmapTimeline/RoadmapTimeline.tsx

```diff
@@ -0,0 +1,206 @@
+import styled from '@emotion/styled';
+import { useMediaQuery } from '@mui/material';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+import MilestoneCard from '../MilestoneCard/MilestoneCard';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+
+const RoadmapTimeline = () => {
+  const { isLight } = useThemeContext();
+  const isMobile = useMediaQuery(lightTheme.breakpoints.down('tablet_768'));
+  const viewAll = false;
+  const milestones = Array.from({ length: 8 });
+
+  const up = milestones.length <= 4 ? milestones : milestones.filter((_, i) => i % 2 === 0);
+  const down = milestones.filter((_, i) => i % 2 !== 0);
+
+  const shouldAddPadding = milestones.length % 2 === 0 && milestones.length > 4;
+
+  return (
+    <div>
+      {isMobile ? (
+        <MobileTimeline isLight={isLight}>
+          <MilestoneCard />
+          <MilestoneCard />
+          <MilestoneCard />
+          <MilestoneCard />
+          <MilestoneCard />
+
+          <ButtonBox>
+            <Button isLight={isLight}>
+              <span>View all Roadmap</span>
```

Comment:
Can you move to style  component

Reply:
In this case, this is used to create content blocks and not to stylize those blocks